runtime/JavaScript/spec/rewriter/generatedCode/abc.js

```diff
@@ -0,0 +1,37 @@
+// Generated from abc.g4 by ANTLR 4.12.0
```

Comment:
this needs to be regenerated during tests or it will fail with a version mismatch